x-pack/plugin/sql/sql-client/src/main/java/org/elasticsearch/xpack/sql/client/ClientVersion.java

```diff
@@ -16,45 +18,17 @@
 import java.util.jar.JarInputStream;
 import java.util.jar.Manifest;
 
-public class Version {
-
-    public static final Version CURRENT;
-    public final String version;
-    public final String hash;
-    public final byte major;
-    public final byte minor;
-    public final byte revision;
-
-    private Version(String version, String hash, byte... parts) {
-        this.version = version;
-        this.hash = hash;
-        this.major = parts[0];
-        this.minor = parts[1];
-        this.revision = parts[2];
-    }
-
-    public static Version fromString(String version) {
-        return new Version(version, "Unknown", from(version));
-    }
+public class ClientVersion {
```

Comment:
Worth adding a javadoc to clarify the difference between `ClientVersion`, `SqlVersion` and `Version`.

Reply:
+1

Reply:
I've added a short description for `SqlVersion` and `ClientVersion`. Hope it's enough, but could elaborate if felt as necessary.